### PR TITLE
add libraries

### PR DESCRIPTION
**Pull request by jenerubia**

### CODE REVIEW

File: .idea/.name [hunk 1]
```diff
@@ -0,0 +1 @@
+On Screen
```
Comment (2212FreaK) [inline .idea/.name]: remove this

File: .idea/codeStyles/Project.xml [hunk 2]
```diff
@@ -0,0 +1,122 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (2212FreaK) [inline .idea/codeStyles/Project.xml]: remove this

File: .idea/codeStyles/codeStyleConfig.xml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (2212FreaK) [inline .idea/codeStyles/codeStyleConfig.xml]: remove this

File: .idea/gradle.xml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (2212FreaK) [inline .idea/gradle.xml]: remove this

File: .idea/libraries/Gradle__androidx_activity_activity_1_0_0_aar.xml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+<component name="libraryTable">
```
Comment (2212FreaK) [inline .idea/libraries/Gradle__androidx_activity_activity_1_0_0_aar.xml]: remove this

File: .idea/libraries/Gradle__androidx_activity_activity_ktx_1_0_0_aar.xml [hunk 6]
```diff
@@ -0,0 +1,12 @@
+<component name="libraryTable">
```
Comment (2212FreaK) [inline .idea/libraries/Gradle__androidx_activity_activity_ktx_1_0_0_aar.xml]: remove this

File: app/build.gradle [hunk 7]
```diff
@@ -0,0 +1,104 @@
+apply plugin: 'com.android.application'
```
Comment (2212FreaK) [inline app/build.gradle]: try using plugin DSL

File: app/build.gradle [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+apply plugin: 'kotlin-android'
+
+apply plugin: 'kotlin-android-extensions'
+
+apply plugin: 'kotlin-kapt'
+
+apply plugin: "androidx.navigation.safeargs.kotlin"
+
+android {
+    compileSdkVersion 29
```
Comment (2212FreaK) [inline app/build.gradle]: move the version to project gradle

File: app/build.gradle [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+
+    dataBinding {
+        enabled = true
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation"org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation 'androidx.appcompat:appcompat:1.0.2'
```
Comment (2212FreaK) [inline app/build.gradle]: home alone?

File: app/build.gradle [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+
+    //ConstraintLayout
+    def constraint_version = "2.0.0-beta3"
```
Comment (2212FreaK) [inline app/build.gradle]: move all versions to project gradle

File: app/build.gradle [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+
+    //Koin
+    def koin_version = '2.0.1'
+    implementation "org.koin:koin-core:$koin_version"
+    // Koin AndroidX Scope features
+    implementation "org.koin:koin-androidx-scope:$koin_version"
+    // Koin AndroidX ViewModel features
+    implementation "org.koin:koin-androidx-viewmodel:$koin_version"
+    // Koin AndroidX Fragment features
+//    implementation "org.koin:koin-androidx-fragment:$koin_version"
```
Comment (2212FreaK) [inline app/build.gradle]: always remove unused code

File: app/src/androidTest/java/com/movie/on/screen/ExampleInstrumentedTest.kt [hunk 12]
```diff
@@ -0,0 +1,24 @@
+package com.movie.on.screen
```
Comment (2212FreaK) [inline app/src/androidTest/java/com/movie/on/screen/ExampleInstrumentedTest.kt]: delete all sample test

File: app/src/main/AndroidManifest.xml [hunk 13]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.movie.on.screen">
+
+    <application
+        android:allowBackup="true"
```
Comment (2212FreaK) [inline app/src/main/AndroidManifest.xml]: false

File: build.gradle [hunk 14]
```diff
@@ -0,0 +1,36 @@
...
+        
+    }
+}
+
+task clean(type: Delete) {
+    delete rootProject.buildDir
+}
+
+
+ext{
```
Comment (2212FreaK) [inline build.gradle:29]: ext{ no space?

File: build.gradle [hunk 15]
```diff
@@ -0,0 +1,36 @@
...
+        def nav_version = "2.1.0"
+        classpath "androidx.navigation:navigation-safe-args-gradle-plugin:$nav_version"
+    }
+}
+
+allprojects {
+    repositories {
+        google()
+        jcenter()
+
```
Comment (2212FreaK) [inline build.gradle]: spacing

File: build.gradle [hunk 16]
```diff
@@ -0,0 +1,36 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    ext.kotlin_version = '1.3.61'
+    repositories {
+        google()
+        jcenter()
+
```
Comment (2212FreaK) [inline build.gradle]: spacing

File: app/src/test/java/com/movie/on/screen/ExampleUnitTest.kt [hunk 17]
```diff
@@ -0,0 +1,17 @@
+package com.movie.on.screen
```
Comment (2212FreaK) [inline app/src/test/java/com/movie/on/screen/ExampleUnitTest.kt:1]: remove this

File: app/build.gradle [hunk 18]
```diff
@@ -0,0 +1,90 @@
...
+    id 'kotlin-android-extensions'
+    id 'kotlin-kapt'
+    id "androidx.navigation.safeargs.kotlin"
+}
+
+android {
+    compileSdkVersion rootProject.compiledVersion
+    buildToolsVersion rootProject.buildToolsVersion
+    defaultConfig {
+        applicationId "com.movie.on.screen"
```
Comment (jenerubia) [inline app/build.gradle]: com.movie.onscreen

File: app/build.gradle [hunk 19]
```diff
@@ -0,0 +1,90 @@
...
+    implementation "androidx.constraintlayout:constraintlayout:$constraintVersion"
+
+    //Material
+    implementation "com.google.android.material:material:$materialVersion"
+
+    //Core KTX
+    implementation "androidx.core:core-ktx:$coreVersion"
+
+    //LiveData
+    implementation "androidx.lifecycle:lifecycle-livedata-ktx:$liveDataVersion"
```
Comment (jenerubia) [inline app/build.gradle]: lifecycleVersion
where is my extension? and kapt

File: app/build.gradle [hunk 20]
```diff
@@ -0,0 +1,90 @@
...
+    //Courotines
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutinesVersion"
+
+    //Room
+    implementation "androidx.room:room-runtime:$roomVersion"
+    kapt "androidx.room:room-compiler:$roomVersion" // For Kotlin use kapt instead of annotationProcessor
+    // optional - Kotlin Extensions and Coroutines support for Room
+    implementation "androidx.room:room-ktx:$roomVersion"
+    // Test helpers
+    testImplementation "androidx.room:room-testing:$roomVersion"
```
Comment (jenerubia) [inline app/build.gradle]: move to bottom

File: app/build.gradle [hunk 21]
```diff
@@ -0,0 +1,90 @@
...
+    implementation "org.koin:koin-androidx-viewmodel:$koinVersion"
+    // Koin AndroidX Experimental features
+    implementation "org.koin:koin-androidx-ext:$koinVersion"
+
+    //MockK
+    testImplementation "io.mockk:mockk:$mockkVersion"
+    androidTestImplementation "io.mockk:mockk-android:$mockkVersion"
+
+    //Testing
+    testImplementation 'junit:junit:4.12'
```
Comment (jenerubia) [inline app/build.gradle]: home alone?

File: app/build.gradle [hunk 22]
```diff
@@ -0,0 +1,90 @@
...
+    kapt "com.github.bumptech.glide:compiler:$glideVersion"
+
+    //Koin
+    implementation "org.koin:koin-core:$koinVersion"
+    // Koin AndroidX Scope features
+    implementation "org.koin:koin-androidx-scope:$koinVersion"
+    // Koin AndroidX ViewModel features
+    implementation "org.koin:koin-androidx-viewmodel:$koinVersion"
+    // Koin AndroidX Experimental features
+    implementation "org.koin:koin-androidx-ext:$koinVersion"
```
Comment (jenerubia) [inline app/build.gradle:81]: remove

**Review by 2212FreaK — APPROVED** [review]

**Pull request merged**